src/core/grid_based_algorithms/LbWalberlaD3Q19MRT.hpp

```diff
@@ -0,0 +1,47 @@
+#include "LbWalberla_impl.hpp"
+#include "MRT_LatticeModel.h"
+
+namespace walberla {
```

Comment:
namespace not following our naming convention in

https://github.com/espressomd/espresso/wiki/Naming-conventions-and-folder-structure#namespaces

Reply:
Not our code, not our naming convention :-) The namespace is part of walberla.
Originally, we had the interface class in an Espresso namespace. However, the use of `using` in class definitions is limited and we ended up with unreadable code due to 100x `walberla::`.